File: serbUber/src/main/java/com/example/serbUber/request/VerifyRequest.java
package com.example.serbUber.request;


import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Positive;

import static com.example.serbUber.exception.ErrorMessagesConstants.WRONG_SECURITY_CODE;

public class VerifyRequest {

    @NotNull(message = WRONG_SECURITY_CODE)
    @Positive(message = WRONG_SECURITY_CODE)
    private Long verifyId;

    @NotNull(message = WRONG_SECURITY_CODE)
    @Positive(message = WRONG_SECURITY_CODE)
    private int securityCode;

    public VerifyRequest(Long verifyId, int securityCode) {
        this.verifyId = verifyId;
        this.securityCode = securityCode;
    }

    public Long getVerifyId() {
        return verifyId;
    }

    public int getSecurityCode() {
        return securityCode;
    }

}

File: angular-app/src/app/model/request/verify-request.ts
export class VerifyRequest {
    verifyId: number;
    securityCode: number;
    userRole: string;

    constructor(
        verifyId: number,
        securityCode: number,
        userRole: string
    ) {
        this.verifyId = verifyId;
        this.securityCode = securityCode;
        this.userRole = userRole;
    }

}